e2e/basic/logs_test.go

```diff
@@ -0,0 +1,97 @@
+package basic
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"time"
+)
+
+// func (s *Suite) XX_TestLogs() {
+// 	const namePrefix = "logs"
+// 	ctx := context.Background()
+
+// 	target, err := s.Knuu.NewInstance(namePrefix + "-target")
+// 	s.Require().NoError(err)
+
+// 	s.Require().NoError(target.Build().SetImage(ctx, alpineImage))
+// 	s.Require().NoError(target.Build().SetStartCommand("sleep", "infinity"))
+// 	s.Require().NoError(target.Build().Commit(ctx))
+// 	s.Require().NoError(target.Execution().Start(ctx))
+
+// 	// Perform the test
+// 	expectedOutput := "Hello World"
+// 	output, err := target.Execution().ExecuteCommand(ctx, "echo", expectedOutput)
+// 	s.Require().NoError(err)
+
+// 	output = strings.TrimSpace(output)
+// 	s.Assert().Equal(expectedOutput, output)
+// }
```

Comment:
Do we need these lines?

Reply:
oh XI, I forgot to uncomment those lines. they test fetching logs for a pod and the next one test fetching logs for a sidecar.